graphene/src/layers/style/mod.rs

```diff
@@ -61,7 +61,7 @@ impl Gradient {
 	}
 
 	/// Adds the gradient def with the uuid specified
-	fn render_defs(&self, svg_defs: &mut String, multiplied_transform: DAffine2, bounds: [DVec2; 2], transformed_bounds: [DVec2; 2]) {
+	fn render_defs(&self, svg_defs: &mut String, multiplied_transform: DAffine2, bounds: [DVec2; 2], transformed_bounds: [DVec2; 2], fill: &Fill) {
```

Comment:
Perhaps it would be better to store the gradient type (currently in `gradient_tool`) as a property of the `Gradient` in the style?

Reply:
That sounds like a good idea! I'll make that change and also move the `GradientType` definition to this file instead since I think that would make more sense.

Reply:
@Hypercube0 we tried adding GradientType as a property to the Gradient struct and a default for GradientType (set to Linear). Despite this, we saw the below error pop up on the Graphite UI while testing our changes, which seems to imply the default isn't being applied to Gradient structs loaded up from existing projects. We're not sure if this is an issue from the Default, Serialize, or Deserialize traits. 
![image](https://user-images.githubusercontent.com/25088940/169098346-52fd6adb-b891-47e0-bec2-701d3565465d.png)

Reply:
@RobNadal This is due to the autosaved document failing to load (due to the json format being changed). Feel free to ignore this.

Reply:
@RobNadal for future reference, that happens commonly when the document serialization format changes from what is saved in your browser between page loads by the auto-save system. You just need to close the open document tab and it'll add a new blank tab that follows the new document format. Nothing to worry about!